RentACar/RentACar/Tests/ClientCRUDTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿using RentACar.Models;
+using RentACar.Repositories;
+using System;
+
+namespace RentACar.Test
+{
+    public static class ClientCRUDTest
+    {
+        static public void TestAll(ClientCRUD clientCrud)
+        {
+            //Create Clients objects
+            var client1 = new Client
+            {
+                Dni = "12345678",
+                Name = "Marcos",
+                LastName = "Mundstock",
+                PhoneNumber = "0123456789",
+                Address = "Av San Martin 123",
+                City = "Carlos Paz",
+                Province = "Cordoba",
+                PostalCode = "5001"
+            };
+
+            var client2 = new Client
+            {
+                Dni = "87654321",
+                Name = "Guillermo",
+                LastName = "Vilas",
+                PhoneNumber = "987654321",
+                Address = "Av Belgrano 987",
+                City = "La Punta",
+                Province = "San Luis",
+                PostalCode = "4009"
+            };
+
+            var client3 = new Client
+            {
+                Dni = "6222111",
+                Name = "Jorge",
+                LastName = "Maronna",
+                PhoneNumber = "1134112345",
+                Address = "Cochabamba 1087",
+                City = "Santo Tome",
+                Province = "Santa Fe",
+                PostalCode = "2001"
+            };
+
+            var client4 = new Client
+            {
+                Dni = "6222111",
+                Name = "Daniel",
+                LastName = "Rabinovich",
+                PhoneNumber = "456789123",
+                Address = "Italia 17",
+                City = "Rosario",
+                Province = "Santa Fe",
+                PostalCode = "2000"
+            };
+
+            //Create Clients in json
+            clientCrud.Create(client1);
+            clientCrud.Create(client2);
+            clientCrud.Create(client3);
+            clientCrud.Create(client4);
+
+            //Get Client from json
+            var getResponse = clientCrud.Get(2);
```

Comment:
ultra minor: response is usually associated with web requests, in this case it's more like a result

Reply:
oh! good tip! i didn't know that.. thanks!